backend/billing/server.go

```diff
@@ -28,18 +28,27 @@ type Access struct {
 	Params map[string]string `json:"params"`
 }
 
+type PortRange struct {
+	Start int `json:"start"`
+	End   int `json:"end"`
+}
+
 type FtpConfig struct {
 	Version        int      `json:"version"`
 	Accesses       []Access `json:"accesses"`
 	Listen_address string   `json:"listen_address"`
+
+	Passive_transfer_port_range PortRange `json:"passive_transfer_port_range"`
```

Comment:
Should we add `TODO: read from configuration file`?

Reply:
I add this TODO in #97